src/app/index.ts

```diff
@@ -0,0 +1,1003 @@
+
+import { get } from 'https';
+import Products from './product';
+
+let products: any = [];
+let loadingQuantity = 4;
+let catalogLength: any = null;
+let filteredProducts: any[] = [];
+const catalogList = document.querySelector(".catalog-list");
+const catalogListPopup = document.querySelector(".catalog-popup");
+const catalogLoadMore = document.querySelector(".catalog__more");
+
+
+class Cart {
+    public goods: any;
```

Comment:
Using `any` removes all type checking provided by TypeScript, which is one major reason for using TypeScript over JavaScript. By using `any`, you expose yourself to difficult to trace and debug issues, especially once the code is deployed in production.
In other words: using `any` for a variable type in TypeScript is similar to writing JavaScript.